packages/expo-development-client/ios/EXDevelopmentClientController.m

```diff
@@ -140,22 +140,33 @@ - (void)loadApp:(NSString *)expoUrl onSuccess:(void (^)())onSuccess onError:(voi
 
 - (void)_initApp:(NSURL *)bundleUrl manifest:(NSDictionary * _Nullable)manifest
 {
-  UIInterfaceOrientationMask orientationMask = UIInterfaceOrientationMaskAll;
-  if ([@"portrait" isEqualToString:manifest[@"orientation"]]) {
-    orientationMask = UIInterfaceOrientationMaskPortrait;
-  } else if ([@"landscape" isEqualToString:manifest[@"orientation"]]) {
-    orientationMask = UIInterfaceOrientationMaskLandscape;
-  }
-  __block UIInterfaceOrientation orientation = [EXDevelopmentClientController defaultOrientationForOrientationMask:orientationMask];
+  __block UIInterfaceOrientation orientation = [EXDevelopmentClientController getOrientationFromManifest:manifest];
+  __block UIColor *backgroundColor = [EXDevelopmentClientController colorWithHexString:[EXDevelopmentClientController readBackgroundColorFromManifest:manifest]];
```

Comment:
As these methods are static, maybe it would make sense to have a separate class for manifest-related utils? 🤔 Maybe we can even wrap manifest dictionary in `EXAppManifest` class that handles all of these

Reply:
Done ;)